

def test_yay():
    raise Exception()